tools/cellprofiler/.gitignore

```diff
@@ -0,0 +1,2 @@
+.venv/
```

Comment:
this should be at the top level :)

Reply:
included in the common .gitignore you mean?

Reply:
yes.